docs/decisions/0002-name-affirmation-plugin.rst

```diff
@@ -0,0 +1,33 @@
+2. Proctoring Email Template Overrides
+--------------------------------------
+
+Status
+------
+
+Accepted
+
+Context
```

Comment:
I think we should give some background on why this was created as a plugin as opposed to other options (IDA, platform app)

Reply:
Agree. Please discuss the product context. Please answer the question: why we need this plugin at all?